client/homebrew/navbar/newbrew.navitem.jsx

```diff
@@ -1,12 +1,95 @@
 const React = require('react');
+const { useState, useRef } = React;
+const _ = require('lodash');
 const Nav = require('naturalcrit/nav/nav.jsx');
+const yaml = require('js-yaml');
 
-module.exports = function(props){
-	return <Nav.item
-		href='/new'
-		newTab={true}
-		color='purple'
-		icon='fas fa-plus-square'>
-		new
-	</Nav.item>;
+const BREWKEY  = 'homebrewery-new';
+const STYLEKEY = 'homebrewery-new-style';
+const METAKEY  = 'homebrewery-new-meta';
+
+const NewBrew = () => {
+
+    
+    const [brew, setBrew] = useState({
```

Comment:
I don't think this is needed either. Once the brew is uploaded, we immediately redirect to `/new`, so there's no need to save it to state.